docs/lasagne.rst

```diff
@@ -54,9 +54,12 @@ API
 
 .. automodule:: nolearn.lasagne
 
-  .. autoclass:: NeuralNet
+  .. autoclass:: NeuralNet(self,layers,**kwargs)
```

Comment:
There was two reasons I did this

1.  The entire signature was repeated twice withing a few lines of itself in the documentation.  Once for the Class doc, and then immediately afterwards for the __init__ method.

2.  At the bottom of the conf.py file, I added some stuff to make it so that the dependency imports are Mocked when Sphinx builds the documentation.  I did this because ReadTheDocsis has trouble installing python packages that use a c backend.  With this in place, some of the defaults are displayed as being an instance of _MockModule.  

ex.  NeuralNet(self, layers, update=<_MockModule @ 0xff21fead234ffed>